nuxt.config.ts

```diff
@@ -56,4 +56,12 @@ export default defineNuxtConfig({
   colorMode: {
     classSuffix: "",
   },
+  // TODO: Option is marked as invalid if env ELECTRON is not enabled, because this enables the module which allows the configuration.
```

Comment:
not sure if I understand this

Reply:
Nuxt generates types, specially those for the valid nuxt-config. If the module `nuxt-electron` is loaded, the config property `electron` is allowed. If its not added (when the env `ELECTRON` is falsy), this property is not valid. Means when running `pnpm dev`, your ide will show it as wrong, same as when running `nuxi typecheck`. You need to set the env `ELECTRON` as truthy for the types to match. See f492f603018691e76e5e47c4ec0a1f0778253185, where this comment was added.